### ISSUE TITLE
Do you not support the GIF format now?

### ISSUE THREAD

**Issue by leoxxw** [mltframework/shotcut#541]
The previous version provided GIF transcode output, now the version is not available, when can I support it?

**Comment by ddennedy** [comment]
What are you making? Are you making a clone of Shotcut and selling it without the source code? Show me what you are doing work Shotcut before I keep helping you without any gain.

**Comment by ddennedy** [comment]
What are you making? Are you making a clone of Shotcut and selling it without the source code? Show me what you are doing with Shotcut before I keep helping you without any gain.

**Comment by ddennedy** [comment]
[Fixed in MLT](https://github.com/mltframework/mlt/issues/284) for next release.

**Comment by leoxxw** [comment]
I used your source code. And made some changes to meet the requirements for me, but I am a novice, a lot of features I wanted, but modify the code level is limited, so I ask you, thank you very much for your help. At present, my program is dependent on your source code, so some functions I am asking you for support. Thank you very much for your help. May I ask if I have learned your code and made changes to implement my own program? May I sell it? If not, how can I get your permission? Thank you very much.
